app/src/app/uploader/page.tsx

```diff
@@ -0,0 +1,148 @@
+'use client';
+import React, {useState} from 'react';
+import Papa from 'papaparse';
+import {ErrorNotification} from '../components/ErrorNotification';
+import {GerryDBViewSelector} from '../components/sidebar/GerryDBViewSelector';
+import {useMapStore} from '@/app/store/mapStore';
+import {Assignment, createMapDocument, patchUpdateAssignments} from '@/app/utils/api/apiHandlers';
+
+type MapLink = {
+  document_id: string;
+  name: string;
+};
+
+export default function Uploader() {
```

Comment:
Q: Should this function be directly available in the dropdown menu on the map page?

Reply:
Agreed!